qiskit/qobj/converters/lo_config.py

```diff
@@ -14,32 +14,36 @@
 
 from qiskit.pulse.channels import DriveChannel, MeasureChannel
 from qiskit.pulse.configuration import LoConfig
+from qiskit.exceptions import QiskitError
 
 
 class LoConfigConverter:
-    """ This class supports to convert LoConfig into ~`lo_freq` attribute of configs.
+    """ This class supports to convert LoConfig into ~ ``lo_freq`` attribute of configs.
     The format of LO frequency setup can be easily modified by replacing
-    `get_qubit_los` and `get_meas_los` to align with your backend.
+    ``get_qubit_los`` and ``get_meas_los`` to align with your backend.
     """
 
-    def __init__(self, qobj_model, qubit_lo_freq, meas_lo_freq,
+    def __init__(self, qobj_model, qubit_lo_freq=None, meas_lo_freq=None,
```

Comment:
Add at end. See comment below about adding keyword arguments to a class.

Reply:
In this case, no positional arguments were added, so I don't think a change is needed

Reply:
It can break code if external users were using keywords arguments as positional arguments, eg, `LoConfigConverter(qobj_model, qubit_lo_range, meas_lo_range)` would break.

Reply:
We discussed offline and agreed this is fine